### DISCUSSION TITLE
Baka Framework

### DISCUSSION THREAD

**Discussion by Gawdzila** [alandtse/fallout_vr_address_library#5] [Ideas]
https://www.nexusmods.com/fallout4/mods/43627?tab=description

This relies on the Address Library, but I can't find a VR version of it.  I've seen a few mods that use this, as it apparently provides "base object swapper" functionality. Any chance this could be converted? It would be nice to build up the availability of these pillars of mod support for VR.

**Comment by FirEmerald** [comment]
I've gotten it to compile replacing the CommonLib it currently uses with a VR-ready CommonLib, but it needs further work and testing before I am sure that it actually works.

**Reply by sii0** [reply]
> I've gotten it to compile replacing the CommonLib it currently uses with a VR-ready CommonLib, but it needs further work and testing before I am sure that it actually works.

by all means please do share if you'd like more people testing :)
